### PR TITLE
 Ensure default cluster dns custom resource when the operator is launched

### PR DESCRIPTION
**Pull request by pravisankar**
- Needed params(currently cluster ip) are fetched from installer config

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/sig-network-edge PTAL

File: manifests/0000_08_cluster-dns-operator_00-custom-resource-definition.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```
Comment (ironcladlou) [inline manifests/0000_08_cluster-dns-operator_00-custom-resource-definition.yaml]: This breaks the resource...

**Comment by ironcladlou** [comment]
Tested the branch with a fix for [the broken CRD](
https://github.com/openshift/cluster-dns-operator/pull/37/files#r225944773) and things work as expected, so will tag once the CRD is fixed.

I'll submit a PR for refactored 4.0 cluster integration test support similar to the one for ingress.

**Comment by pravisankar** [comment]
@ironcladlou updated, PTAL

**Comment by ironcladlou** [comment]
Please squash then I'll tag

**Comment by pravisankar** [comment]
@ironcladlou squashed related commits

**Comment by pravisankar** [comment]
Tests failed with this error:
```
 aws_eip.nat_eip.5: Error creating EIP: AddressLimitExceeded: The maximum number of addresses has been reached.
```
/retest

**Pull request merged**